docs/source/home/install.rst

```diff
@@ -44,9 +46,28 @@ Create Tables
     Refer to the CLI command `create <../cli/cli_create.ipynb>`_, `populate <../cli/cli_populate.ipynb>`_ and
     `init <../cli/cli_init.ipynb>`_ for more detailed information.
 
-After the database has been installed, you can use the `metacatalog CLI <../cli/cli.rst>`
-to create the necessary tables:
+After the database has been installed, you can use the `metacatalog CLI <../cli/cli.rst>`_
+to create the necessary tables.
+Follow the syntax of the init command and replace *driver*, *user*, *password*, *host* and *database* with your parameters
+
+.. note::
+
+    All parameters can be accessed in pgAdmin or have been previously set by the user.
+
+.. code-block:: bash
+
+    metacatalog init --connection driver://user:password@host:port/database
+
+When using Windows this command can lead to errors and must be changed in this case (refer to `metacatalog CLI <../cli/cli.rst>`_):
+
+.. code-block:: bash
+
+    python -m metacatalog init --connection driver://user:password@host:port/database
+
+
+
+The (standard) connection command could look like this:
 
 .. code-block:: bash
 
-    metacatalog init --connection postgresql://user:password@host:port/dbname
+    python -m metacatalog init --connection postgresql://postgres:\ *yourpassword*\ @localhost:5432/metacatalog
```

Comment:
I think I sticked to omitting the `python -m` part, as on Linux it is available like that. My plan was always to fix this behavior on Windows (To work there as well as a shell script). But I think there is not really enough time for this and it is of really low proirity. So, if we merge like this, we need to change the CLI calls everywhere in the docs. Otherwise it becomes inconsistent.